Gradle_Build_Tool_Configuration_Organization/demos/build-src/settings.gradle.kts

```diff
@@ -10,7 +10,7 @@
 plugins {
     // Apply the foojay-resolver plugin to allow automatic download of JDKs
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.4.0"
-    id("com.gradle.enterprise") version "3.13.2"
+    id("com.gradle.enterprise") version "3.16.2"
 }
 
 gradleEnterprise {
```

Comment:
Some of these settings files are configured to publish build scans to scans.gradle.com.  Do we want to update these to publish to the DPE Univerisity Develocity instance instead?

Reply:
Yes - but not at this time. Only the caching training courses are going to be available in the self-paced platform at this time. We can update the exercises for the other training courses when they are available on the self-paced platform.